hack/package/package.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# VERSION the version to give to the package
+VERSION=$1
```

Comment:
use VERSION=${1:?"Version must required"}  to give error message is the package.sh script is not used correctly. The same for the following two variables.